packetbeat/route/route_windows.go

```diff
@@ -0,0 +1,198 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//nolint:unused,structcheck // How many ways to check for unused? (╯°□°）╯︵ ┻━┻ Fields kept for documentation.
+package route
+
+import (
+	"errors"
+	"runtime"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+var (
+	// For details of the APIs used, see:
+	// https://docs.microsoft.com/en-us/windows/win32/api/iphlpapi/nf-iphlpapi-getbestinterfaceex
+	// https://docs.microsoft.com/en-us/windows/win32/api/iphlpapi/nf-iphlpapi-getadaptersaddresses
+	libiphlpapi          = windows.NewLazySystemDLL("Iphlpapi.dll")
+	getBestInterfaceEx   = libiphlpapi.NewProc("GetBestInterfaceEx")
```

Comment:
These would be good candidates for github.com/elastic/go-windows if you wanted to go through the effort.

Reply:
I'll file an issue on that repo to migrate/copy these to there.